InterfacePython/TasmanianSG.py

```diff
@@ -713,6 +714,63 @@ def makeFourierGrid(self, iDimension, iOutputs, iDepth, sType, liAnisotropicWeig
 
         pLibTSG.tsgMakeFourierGrid(self.pGrid, iDimension, iOutputs, iDepth, c_char_p(sType), pAnisoWeights, pLevelLimits)
 
+    def makeGridFromCustomTabulated(self, iDimension, iOutputs, iDepth, sType, iCustomTabulated, liAnisotropicWeights=[], liLevelLimits=[]):
```

Comment:
* the name is inconsistent, call this `makeGlobalGridCustom()`, Global grid is a type in itself that exists in documentation and we should be explicit that this is a different way to make the grid as opposed to a new grid type
* no need to copy-paste the documentation, refer to the inputs of `makeGlobalGrid()` and outline the difference, that way if we have to update the docs, we won't have to update them in two places (or forget one place)
* procedurize the error checking, make new method `_testMakeGlobalGrid()` and add the error checking conditions for the common parameters in one place (I should have done that with the other grid methods)
* note the use of underscore in front of a method that indicates the method is not intended to be called by the user, similar to `private` or `protected` in C++
*